server/pbenchinacan/etc/pbench-server/pbench-server.cfg

```diff
@@ -32,16 +32,19 @@ db_uri = postgresql://pbenchcontainer:pbench@pbenchinacan/pbenchcontainer
 [authentication]
 
 # URL of the OIDC auth server with port
-server_url = keycloak.example.com:0000
+server_url = http://localhost:8090
+
+# URL of the OIDC auth server with port for pbench server service
+internal_server_url = http://localhost:8080
```

Comment:
I don't think you need to include "with port" -- ports are permissible in URLs, and specifying them is necessary if addressing a port other than 80 with the `http://` schema.

However, the rest of the text doesn't tell me very much.  What is this value used for, and/or how is it different from the `server_url` value?  (At the moment, I don't see any references to this definition...are those changes supposed to be in this PR?)

Reply:
I will add some description in the server config file.

I was thinking to add the references to this definition (`internal_server_url`) in PR #3056 where we are initializing the OIDC auth client. However, your comment made me realize that I need to add a simple health check for the Keycloak server inside the pbench-server `shell.py` and that should be a part of this PR. So the health check will be using this variable if present otherwise will use the regular `server_url`.